status.html

```diff
@@ -1,124 +1,84 @@
-                    <span class="component dev-ready">Complete</span>
-                  {%- elsif page.status == "deprecated" -%}
-                    <span class="component deprecated">Deprectated</span>
-                  {%- endif -%}
-                {%- else -%}
-                  <span class="component prototype">In Progress</span>
-                {%- endif -%}
+                <td class="align-center">
+                {% if page.status %}
+                  <span class="status-label small {{page.status | replace: ' ','-'}}">{{page.status}}</span>
+                {% endif %}
+                </td>
+                <td class="align-center status-ok">
+                  {% if page.helix-ui-css %}
+                  <hx-icon type="checkmark-circle"></hx-icon>
+                  {% endif %}
+                </td>
+                <td class="align-center status-ok">
+                  {% if page.helix-ui-javascript %}
+                  <hx-icon type="checkmark-circle"></hx-icon>
+                  {% endif %}
                 </td>
               </tr>
-          {%- endif -%}
-      {%- endfor -%}
-    </tbody>
-    </table>
-<br>
-<br>
-
-    <h2>Option 2</h2>
-
-    <div class="row" style="max-width: 40em;">
-    <div class="col-md-3"><i class="icon-disabled"></i><p>Unavailable</p></div>
-    <div class="col-md-3"><i class="icon-warning"></i><p>In-Progress</p></div>
-    <div class="col-md-3"><i class="icon-check-mark"></i><p>Complete</p></div>
-    <div class="col-md-3"><i class="icon-error"></i><p>Deprecated</p></div>
-  </div>-->
-  <h2>Component Status</h2>
-  <table class="striped component-table">
-    <thead>
-      <tr class="compressed">
-        <th class="th-1-2">Component</th>
-        <!-- Removing this line until we have a strategy around versioning as this is a future item<th class="th-1-3">Version Added</th> -->
-        <th class="th-1-2">Status</th>
-      </tr>
-    </thead>
-    <tbody>
-      {%- for page in site.pages -%}
-          {%- if page.category -%}
-              <tr class="compressed">
-                <td class="compressed"><a href="{{site.baseurl}}{{page.url}}.html">{{page.title}}</a></td>
-                <!-- Removing this line until we have a strategy around versioning as this is a future item
-                <td class="compressed">1.0</td> -->
-                <td class="compressed">
-                {%- if page.status -%}
-                  {%- if page.status == "pending" -%}
-                    <i class="icon-warning"></i>
-                  {%- elsif page.status == "complete" -%}
-                    <i class="icon-check-mark"></i>
-                  {%- elsif page.status == "deprecated" -%}
-                    <i class="icon-error"></i>
-                  {%- endif -%}
-                {%- else -%}
-                  <i class="icon-disabled"></i>
-                {%- endif -%}
+          {% endif %}
+      {% endfor %}
+      {% for page in site.component_design %}
```

Comment:
Minor nit: It's a bit odd to have the alphabetical listing restart here. Can we add a table spanner above each block that says something like "General Patterns" for the docs and "Component Patterns" for the components?